### PR TITLE
fix: add config-file for github plugin (#218)

### LINKED ISSUE #218: Plugin Shows in Goland Marketplace, but Fails to Install
The plugin shows in the Marketplace for Goland, but when trying to install it fails. Please see screen shots below

image

### PR DESCRIPTION
**Pull request by lstocchi**
it resolves #218

### CODE REVIEW

File: src/main/resources/META-INF/plugin-github.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<idea-plugin>
+    <extensions defaultExtensionNs="com.intellij">
+        <applicationService serviceImplementation="com.redhat.devtools.intellij.knative.git.GitHandler"/>
```
Comment (jeffmaury) [inline src/main/resources/META-INF/plugin-github.xml:3]: I don't get it, why do you declare them as service if you don't use them as such ?

Comment (lstocchi) [reply src/main/resources/META-INF/plugin-github.xml:3]: No valid reason, just a lack of alternatives. I don't really need anything as I check if the gh plugin is installed and I disable the action myself. So I used services as they are loaded on demand so if there is no gh plugin they are skipped. Which tag should i use?

Comment (jeffmaury) [reply src/main/resources/META-INF/plugin-github.xml:3]: Can't you simply have an empty file ?

Comment (lstocchi) [reply src/main/resources/META-INF/plugin-github.xml]: :expressionless: ... done

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**